crates/revm/src/state_change.rs

```diff
@@ -51,11 +55,131 @@ pub fn post_block_balance_increments(
     balance_increments
 }
 
-/// Applies the pre-block call to the EIP-4788 beacon block root contract, using the given block,
+/// todo: temporary move over of constants from revm until we've migrated to the latest version
+pub const HISTORY_SERVE_WINDOW: usize = 8192;
+
+/// todo: temporary move over of constants from revm until we've migrated to the latest version
+pub const HISTORY_STORAGE_ADDRESS: Address = address!("25a219378dad9b3503c8268c9ca836a52427a4fb");
```

Comment:
Parameter | Value
-- | --
FORK_TIMESTAMP | TBD
HISTORY_STORAGE_ADDRESS | 0x25a219378dad9b3503c8268c9ca836a52427a4fb
HISTORY_SERVE_WINDOW | 8192
BLOCKHASH_OLD_WINDOW | 256

from the https://eips.ethereum.org/EIPS/eip-2935, seems OK?